main/lxc/lxc.initd

```diff
@@ -43,31 +46,52 @@ checkconfig() {
 	# no need to output anything, the function takes care of that.
 	[ -z "${CONFIGFILE}" ] && return 1
 
-	utsname=$(lxc_get_var lxc.utsname)
+	utsname=$(lxc_get_var lxc.uts.name)
 	if [ "${CONTAINER}" != "${utsname}" ]; then
 	    eerror "You should use the same name for the service and the"
-	    eerror "lxc.utsname. Right now the lxc.utsname is set to ${utsname}"
+	    eerror "lxc.uts.name : Right now the lxc.uts.name is set to : ${utsname}"
 	    return 1
 	fi
 }
 
+systemd_ctr() {
+	# required for lxc-console & services inside systemd containers
+	local cgroup=/sys/fs/cgroup/systemd cmd=$1
+	local mnt_opts='rw,nosuid,nodev,noexec,relatime,none,name=systemd'
+
+	case "$cmd" in
+		 mount) checkpath -d $cgroup
+			if ! mount | grep $cgroup 1>/dev/null; then
+				mount -t cgroup -o $mnt_opts cgroup $cgroup
+			fi
+			;;
+	       unmount) if mount | grep $cgroup 1>/dev/null; then
+				umount $cgroup
+			fi
+			;;
+	esac
```

Comment:
Please use tabs for indentation, not spaces.

If you use vim, install [vim-editorconfig](https://pkgs.alpinelinux.org/package/edge/community/x86_64/vim-editorconfig). There’s editorconfig plugin for basically all relevant editors.

Reply:
it's tabs with a few spaces so the case statements line up - rather than only tabs & being uneven

Reply:
with only tabs:
```
        case "$cmd" in
                mount) checkpath -d $cgroup
                        if ! mount | grep $cgroup 1>/dev/null; then
                                mount -t cgroup -o $mnt_opts cgroup $cgroup
                        fi
                        ;;
                unmount) if mount | grep $cgroup 1>/dev/null; then
                                umount $cgroup
                        fi
                        ;;
        esac
```
as it currently is with mount / unmount level with each other:
```
        case "$cmd" in
                 mount) checkpath -d $cgroup
                        if ! mount | grep $cgroup 1>/dev/null; then
                                mount -t cgroup -o $mnt_opts cgroup $cgroup
                        fi
                        ;;
               unmount) if mount | grep $cgroup 1>/dev/null; then
                                umount $cgroup
                        fi
                        ;;
        esac
```

Reply:
That’s wrong alignment. Simple test: change tab width in your editor and look how it gets broken.

When you use tabs (as we do in shell scripts, e.g. to make `<<-EOF` usable), it’s better to resign to alignment with spaces and stick only to tabs (at least in community/team project), because most people can’t (or don’t care to) do it right. And so it quickly becomes total mess. Actually that’s the main reason why most langs/projects prefer spaces for both indentation and alignment.